### PR TITLE
tox test for multiple python versions

### PR DESCRIPTION
**Pull request by hirosassa**
This PR is for #168 and #157.
I implemented `tox` configuration for testing on multiple python versions and introduced `tox-gh-actions`.

### CODE REVIEW

**Comment by hirosassa** [comment]
@Hi-king @vaaaaanquish plz review!

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,21 +7,23 @@ on:
...
-  build:
-
-    runs-on: ubuntu-latest
+  tests:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 2
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.7, 3.8]
```
Comment (Hi-king) [inline .github/workflows/test.yml:16]: @hirosassa Why not adding 3.9 ? :)

Comment (hirosassa) [reply .github/workflows/test.yml]: @Hi-king thanks, added!

**Review by Hi-king — APPROVED** [review]
Great! Thx

**Comment by vaaaaanquish** [comment]
Thx!!

**Review by vaaaaanquish — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (hirosassa) [reply .github/workflows/test.yml]: @Hi-king `xml.etree.ElementTree.getchildren` is deleted on python 3.9 and test fails.
IMO, skip to support python 3.9 in this PR, and will tackle on next.

https://github.com/m3dev/gokart/runs/1788285866?check_suite_focus=true

Comment (Hi-king) [reply .github/workflows/test.yml:16]: Oh...This really is problem 😿

**Pull request merged**